CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   * Removed version cap on xarray
 * Bug Fix
    * Improved compatibility with xarray 2022.06 when accessing data via slices
+   * Fixed a bug for fractional seconds in `methods.generate_times`.
```

Comment:
```suggestion
   * Fixed a bug for fractional seconds in `methods.testing.generate_times`.
```

Reply:
Looks complete to me once this commit is merged.